modules/ROOT/pages/clustering/clustering-advanced/reconciler.adoc

```diff
@@ -0,0 +1,58 @@
+[role=enterprise-edition]
+[[cluster-reconciler]]
+= Reconciler
+:description: This section describes how changes to the DBMS are processed by each server.
+
+[[cluster-reconciler-introduction]]
+== Introduction
+
+Administrative operations in Neo4j, such as creating a database, do not happen synchronously.
+Instead we have a model where the changes are written to the `system` database, and record the desired state of the DBMS.
+Each server has an internal component called the reconciler, which observes the desired state and makes changes to the local server to match that state.
+For example, the `system` database may record that it wants the database `bar` running on `server-1`.
+When the reconciler on `server-1` sees that, it will start `bar`.
+
+Executing an operation and getting back a successful response means that the desire is safely committed to the `system` database, and will be processed by each member of the cluster at some point, assuming the server is healthy enough.
+
+[[cluster-reconciler-async]]
+== Asynchronicity
+Servers can observe the same operation at different times, since the `system` database is just another Raft group, where followers and replicas can lag behind the leader in some situations.
+Each server should eventually see the new state though, and take action on it.
```

Comment:
This remark leaves readers with doubt. _Are there circumstances where the new state is not propagated? Do I need to be careful and account for it? Are there known issues on this?_
Can we make it affirmative, as in 

> At all rates/Sooner or later, the new state gets eventually propagated to all servers.

Reply:
The doubt is sort-of deliberate. There _are_ circumstances where a server will not see the results, though they are examples of bad problems. The server being permanently offline, long network partitions etc.

I could add a different qualifier, like:
> Eventually, the new state gets propagated to all healthy servers.

Reply:
> Eventually, the new state gets propagated to all healthy servers.

This sounds great.